### ISSUE TITLE
fix 99 links with temporary redirects

### ISSUE THREAD

**Issue by susanev** [pulumi/registry#2287]
we have 99 links with temporary redirects, the issue here is that links need to end in `/`
some of these can be fixed easily directly in the registry repo
others are going to require prs in other repos
I'm not sure why basically every azure  link is failing now...

[internal spreadsheet](https://docs.google.com/spreadsheets/d/1sPnAqVf6AyTPGn28kBjdOppkJ_A6s8dwG8kCqlVDzt8/edit?usp=sharing)

@sean1588 thoughts on being able to lint and automatically fix these?

**Comment by tofuni** [comment]
Hey @susanev and @sean1588! Looping back here with the [updated crawl on temporary redirects](https://docs.google.com/spreadsheets/d/1nqKEpbGNDouzGRcpZONAbchhtl0AyooP0zVNqQMsUQA/edit#gid=0). 

It looks like our suspicions were right on it being a more pervasive, site-wide issue - so the linting solution is probably the ideal way to go still.

**Comment by sean1588** [comment]
@tofuni yeah agreed, I think going about fixing this in an automated way would be most ideal. thanks for sharing this!

**Comment by cnunciato** [comment]
@sean1588 Should this issue be renamed and updated to be about adding a general lint (or auto-correct) step for missing trailing slashes?
